CODE_OF_CONDUCT.md

```diff
@@ -2,48 +2,119 @@
-- We respect that people have differences of opinion and criticize constructively.
+- [contributor-covenant.org/translations](https://www.contributor-covenant.org/translations)
+- [github.com/ContributorCovenant](https://github.com/ContributorCovenant/contributor_covenant/tree/release/content/version/2/0)
 
-## Don't
+Refer to the sections on reporting and escalation in this document for the specific emails that can be used to report and escalate issues.
 
-- Don’t be mean or rude.
-- Don’t discriminate against anyone. Although we have phrased the formal diversity statement generically to make it all-inclusive, we recognize that there are specific attributes that are used to discriminate against people. In alphabetical order, some of these attributes include (but are not limited to): age, culture, ethnicity, gender identity or expression, national origin, physical or mental difference, politics, race, religion, sex, sexual orientation, socio-economic status, and subculture. We welcome people regardless of these or other attributes.
-- Sexism and racism of any kind (including sexist and racist “jokes”), demeaning or insulting behaviour and harassment are seen as direct violations to this code of conduct. Harassment includes offensive verbal comments related to gender, sexual orientation, disability, physical appearance, body size, race, religion, sexual images in public spaces, deliberate intimidation, stalking, following, harassing photography or recording, inappropriate physical contact, and unwelcome sexual attention.
-- On IRC, Slack and other online or offline communications channels, don't use overtly sexual nicknames or other nicknames that might detract from a friendly, safe and welcoming environment for all.
-- Unwelcome / non-consensual sexual advances over IRC or any other channels related with this community are not okay.
-- Derailing, tone arguments and otherwise playing on people's desires to be nice are not welcome, especially in discussions about violations to this code of conduct.
-- Please avoid unstructured critique.
-- Likewise, any spamming, trolling, flaming, baiting or other attention-stealing behaviour is not welcome.
+## Reporting
 
-## Consequences for violations of this code of conduct
+### AMP Project Spaces
 
-If a participant engages in any behavior violating this code of conduct, the [Code of Conduct Working Group](https://github.com/ampproject/wg-codeofconduct) may take any action they deem appropriate, including warning the offender or expulsion from the community, exclusion from any interaction and loss of all rights in this community.
+For reporting issues in spaces related to a member project please use the email provided by the project for reporting. Projects handle CoC issues related to the spaces that they maintain. Projects maintainers commit to:
```

Comment:
For someone not familiar with OpenJSF who just wants to figure out who they should send an email to, it's not clear  how they would find "the email provided by the project."  I know it's in the next section, but that can be confusing (TBH, I'm a little confused by the split here :)).

Can we inline the address here like we do below?

Reply:
Good point added inline. Please take a look?